src/path.cr

```diff
@@ -516,20 +488,19 @@ struct Path
 
   private def each_part_separator_index(offset = 0)
     reader = Char::Reader.new(@name)
+    offset = 0
     reader.pos = start_pos = offset
```

Comment:
Even that offset variable and put zero here?

Reply:
Yes, that's just a leftover from a step in between